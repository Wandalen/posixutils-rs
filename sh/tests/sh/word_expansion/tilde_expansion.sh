HOME=/home/test

echo ~
echo ~/dir
a=~/a:b:~/c
echo $a