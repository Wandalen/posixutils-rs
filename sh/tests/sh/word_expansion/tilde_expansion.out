/home/test
/home/test/dir
/home/test/a:b:/home/test/c
